Source/FieldSolver/WarpX_QED_Field_Pushers.cpp

```diff
@@ -0,0 +1,169 @@
+#include <cmath>
+#include <limits>
+
+#include <WarpX.H>
+#include <WarpXConst.H>
+#include <WarpX_f.H>
+#include <WarpX_K.H>
+#include <WarpX_PML_kernels.H>
+#include <WarpX_FDTD.H>
+#ifdef WARPX_USE_PY
+#include <WarpX_py.H>
+#endif
+
+#include <WarpX_QED_K.H>
+
+#include <PML_current.H>
+
+#ifdef BL_USE_SENSEI_INSITU
+#include <AMReX_AmrMeshInSituBridge.H>
+#endif
+
+using namespace amrex;
+
+
+void
+WarpX::Hybrid_QED_Push (Real a_dt)
```

Comment:
Could you pass the whole `dt` vector here, and then `dt[lev]` below in the call to `Hybrid_QED_Push`?